src/interfaces/business-interfaces.ts

```diff
@@ -5,6 +5,7 @@ export interface FilingTypeIF {
   displayName: string
   feeCode: string
   name: FilingTypes
+  type?: string // FUTURE: use an enum
```

Comment:
There are only 4 subtypes at the moment (all related to restorations). There may be more values for dissolution types. When we know more (future) then we can improve this.